config/routes.rb

```diff
@@ -20,7 +20,7 @@
   end
 
   get "/autoyast", to: "dashboard#autoyast"
-  get "/kubectl-config", to: "oidc#index"
+  get "/kubeconfig", to: "oidc#index"
```

Comment:
We have already said to customers that they could use `/kubectl-config`, I would leave that path as it is. If you want to introduce `/kubeconfig` we should keep `/kubectl-config` as a redirect to `/kubeconfig`, so we don't break the old behavior.